bears/r/RLintBear.py

```diff
@@ -28,6 +28,7 @@ class RLintBear:
                     DistributionRequirement(apt_get='r-base',
                                             version='>=3.1.1')}
     LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/FIq6VmrcdruXdgdGxwG9DdyuD'
```

Comment:
asciinema is not public

Reply:
@jayvdb I've updated the link. Please review.